aws/PAS-network-environment-NAT.json

```diff
@@ -1026,6 +1069,392 @@
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGIngress15": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "udp",
+                "FromPort": 514,
+                "ToPort": 11514,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress1": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 443,
+                "ToPort": 443,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress2": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 514,
+                "ToPort": 514,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress3": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "udp",
+                "FromPort": 514,
+                "ToPort": 514,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress4": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 80,
+                "ToPort": 80,
+                "DestinationSecurityGroupId": {
+                    "Ref" : "PVWASG"   
+                }
+            }
+        },
+        "PTASGEgress5": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "udp",
+                "FromPort": 53,
+                "ToPort": 53,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress6": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "udp",
+                "FromPort": 123,
+                "ToPort": 123,
+                "CidrIp": "0.0.0.0/0"
+            }
+        },
+        "PTASGEgress7": {
+            "Type": "AWS::EC2::SecurityGroupEgress",
+            "Properties": {
+                "GroupId": {
+                    "Ref": "PTASG"
+                },
+                "IpProtocol": "tcp",
+                "FromPort": 27017,
```

Comment:
@maishsk , do we really need to open all these ports or they are relevant only per application?
If it is only per application I think we need to mention in documentation that in case customer wants to monitor application he will need to open these ports manually, but we should keep them closed by default.
The same as per CPM.
@laviebar  ?